source/common/stream_info/utility.h

```diff
@@ -98,6 +98,7 @@ class TimingUtility {
   absl::optional<std::chrono::nanoseconds> lastDownstreamTxByteSent();
   absl::optional<std::chrono::nanoseconds> lastDownstreamRxByteReceived();
   absl::optional<std::chrono::nanoseconds> downstreamHandshakeComplete();
+  absl::optional<std::chrono::nanoseconds> upstreamHandshakeComplete();
```

Comment:
small nit: Can you move this function to the line 96? so that upstream and downstream utilities are grouped together as before?

Same for function definition.

Reply:
Sure, done.